packages/syft/src/syft/service/request/request.py

```diff
@@ -290,21 +330,13 @@ def accept_by_depositing_result(self, result: Any, force: bool = False):
                 apply_permission_type=ActionPermission.READ,
             )
 
-            submit_request = SubmitRequest(
-                changes=[policy_state_mutation, permission_change],
-                requesting_user_verify_key=self.requesting_user_verify_key,
-            )
-
-            self.approve()
+            new_changes = [policy_state_mutation, permission_change]
+            result = api.services.request.add_changes(uid=self.id, changes=new_changes)
```

Comment:
Adding the new set of changes to the same request, so that we can easily track what are all the changes that took place w.r.t a particular request. This is useful when we revert changes.